bot/cogs/free.py

```diff
@@ -80,7 +79,8 @@ class Free(Cog):
             # Sort channels in descending order by seconds
             # Get position in list, inactivity, and channel object
             # For each channel, add to embed.description
-            for i, (inactive, channel) in enumerate(sorted(free_channels, reverse=True), 1):
+            sorted_channels = sorted(free_channels, key=lambda free_channel: free_channel[0], reverse=True)
```

Comment:
Any reason for using a `lambda` instead of `operator.itemgetter(0)` here? If you `import itemgetter from operator`, we wouldn't need a lambda function: 

```suggestion
            sorted_channels = sorted(free_channels, key=itemgetter(0), reverse=True)
```

Reply:
Well, I don't care either way but one reason to use a lambda is because nothing needs to be imported.

Reply:
Is there a specific benefit to using the itemgetter vs not needing the import and using a lambda?  And honestly the reason I did the lambda was because I forgot about that itemgetter

Reply:
Personally, I think that itemgetter here is more idiomatic and lambdas shouldn't be used in these types of situations. That said, the channel lists are relatively small, so the performance benefit of itemgetter does not make a significant difference for us. I do think it's an improvement for the code style, though.

Reply:
That's fair.  Yeah for the sake of consistent style I'll swap it over